src/herbpy/action/actions.py

```diff
@@ -0,0 +1,145 @@
+import numpy, openravepy, prpy, time
+from prpy import tsr
+from prpy.tsr import tsrlibrary
+from prpy.tsr.tsr import *
+
+@ActionMethod
+def MoveObject(robot, direction = [0, 0, 1], distance = 0.1):
+	"""
+	@param robot The robot performing the push grasp
+	@param direction The direction to move the object
+	@param distance The distance to move the object
+	"""
+    try:
+        manip = robot.GetActiveManipulator()
+        manip.PlanToEndEffectorOffset(direction, distance, position_tolerance=0.1, execute=True)
+        return True
+    except Exception, e:
+        print 'MoveObject planning failed: ', str(e)
+        return False
```

Comment:
We shouldn't `print` in library functions - use a logger instead.

However, in this case, I think it would be better to avoid catching the exception all-together. I believe we typically indicate failure in an action by raising a `PlanningError` or `ActionError`; not returning `False`. @jeking04 can you confirm?